saleor/order/interface.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+from decimal import Decimal
+
+from saleor.payment.models import Payment
+
+
+@dataclass
+class OrderPaymentAction:
```

Comment:
IMO no. The reason is that this dataclass is passed to functions that are already operating in the context of a particular payment action. It would make it extra annoying to use that class and duplicate the action type every single time.